packages/adapter-fauna/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@auth/fauna-adapter",
-  "version": "1.3.2",
+  "version": "2.0.0",
```

Comment:
As discussed in this comment https://github.com/nextauthjs/next-auth/issues/8394#issuecomment-1690655534, we should make a major version bump for the package. People who previously used the adapter need to rename their Fauna collections to make 2.0 work. 
I also considered accepting a config in the adapter, specifying what the collection names are. These could then be used with string interpolation, so people can use any naming schema they want for their collections (or keep the old ones). Let me know what you think.

Reply:
If you think the collection names config sounds like a good idea, I'd like to open a new PR for this after we get this merged